server/location/requests.js

```diff
@@ -0,0 +1,163 @@
+
+const PostcodesIO = require('postcodesio-client');
```

Comment:
i don't think we should have two libraries doing the same thing. this one (-client) seems to be more popular, so we could just use this one?